recipes/qt/5.x.x/conanfile.py

```diff
@@ -256,6 +256,7 @@ def _enablemodule(mod):
                 _enablemodule(module)
 
     def validate(self):
+        tools.check_min_cppstd(self, "11")
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-is-a-toolscheck_min_cppstd-call-not-enough

Reply:
check_min_cppstd is already guarded against empty cppstd: https://github.com/conan-io/conan/blob/f230bf104842c160be6ea377497151fe1ea64c38/conans/client/tools/settings.py#L41 and in this case it effectively uses the actual compiler's default for std.
If we add this `if self.settings.compiler.get_safe("cppstd")`, then the users using default configuration without explicit cppstd setting (most users) will not be stopped by this error, and then they get a cryptic error message when they try to consume the project. I had to help several users with this situation.

Reply:
We had previously sided that we should block those unsupported by default compilers.
I agree about the behavior but perhaps we should go about it differently?

Reply:
OK, this is controversial, and I don't want to delay the integration of the other changes, so here we go